libs/framework/middlewares/src/render.test.tsx

```diff
@@ -49,7 +49,7 @@ jest.useFakeTimers()
 
 describe('render middleware', () => {
   beforeEach(() => {
-    renderToPipeableStream.mockImplementation((el, obj) => {
+    ;(renderToPipeableStream as jest.Mock).mockImplementation((el, obj) => {
```

Comment:
is this a typo?

Reply:
This is probably a typo. But most likely a side affect of prettier not using semicolons. As the casting could be applied as function arguments on the previous line